.docker/web/Dockerfile

```
@@ -0,0 +1,36 @@
+WORKDIR /app
+COPY --from=builder /usr/local/bin /usr/local/bin
+COPY --from=builder /usr/local/lib/python3.11/site-packages /usr/local/lib/python3.11/site-packages
+ENV FLASK_APP=app.py \
+    FLASK_ENV=development \
+    FLASK_RUN_PORT=5000 \
+    FLASK_RUN_HOST=0.0.0.0
+EXPOSE 5000
+CMD ["flask", "run"]
+
+FROM base AS gunicorn
+RUN pip install --no-cache-dir gunicorn
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)